packages/cli/scripts/bootstrapFinder.ts

```diff
@@ -0,0 +1,84 @@
+import jayson from 'jayson/promise/index.js'
+
+import { createBeaconConfig, defaultChainConfig, BeaconConfig } from '@lodestar/config'
+import { genesisData } from '@lodestar/config/networks'
+import { BeaconLightClientNetworkContentType, fromHexString, getBeaconContentKey, LightClientBootstrapKey, LightClientOptimisticUpdateKey, LightClientUpdatesByRange, LightClientUpdatesByRangeKey, ProtocolId, toHexString } from 'portalnetwork'
+
+import { ssz } from '@lodestar/types'
+import { ForkName } from '@lodestar/params'
+import { computeSyncPeriodAtSlot } from '@lodestar/light-client/utils'
+import { concatBytes, hexToBytes } from '@ethereumjs/util'
+
+
+const { Client } = jayson
+
+const main = async () => {
+    const beaconConfig = createBeaconConfig(defaultChainConfig, hexToBytes(genesisData.mainnet.genesisValidatorsRoot))
+    const capellaForkDigest = beaconConfig.forkName2ForkDigest(ForkName.capella)
+    const beaconNode = 'https://lodestar-mainnet.chainsafe.io/'
+    const ultralights: jayson.HttpClient[] = []
+    for (let x = 0; x < 10; x++) {
+        ultralights.push(Client.http({ host: '127.0.0.1', port: 8545 + x }))
+    }
+
+
+    console.log('Retrieving bootstrap and updates from Beacon node...')
+    let optimisticUpdate = ssz.capella.LightClientOptimisticUpdate.fromJson((await (await fetch(beaconNode + 'eth/v1/beacon/light_client/optimistic_update')).json()).data)
```

Comment:
this can become more convinient if you use `@lodestar/api`:
```typescript
      import {getClient} from "@lodestar/api"
      const api = getClient({baseUrl}, {config});
      api.lightclient.getUpdates(...)
      api.lightclient.getOptimisticUpdate(...)
      ```